docs/intro.md

```diff
@@ -0,0 +1,58 @@
+### Quickstart
+
+Assuming you have all neccessary requirements installed you can start up the entire project using either GNU Make:
+
+```bash
+make start
+```
+
+or bash
+
+```bash
+./manage start # or ./manage up
+```
+
+in the root of the project.
+
+You can stop the the project by running
+
+```bash
+./manage down # /.manage stop
+```
+
+or
+
+```bash
+make stop
+```
+
+You can rid yourself of the spun-up containers by using
+
+```bash
+make stop_n_clean
+```
+
+### System Requirements
+
+A machine with:
+
+- docker
+- docker-compose
+- Bash
+- GNU Make (optional)
+
+You probably want at least 5GB of free disk space, 4GB of RAM and an average CPU like an Intel i5.
+
+### Components and tweaking
+
+The standard setup spins up the the entire infrastructure required with the following containers having Swagger interfaces:
+
+- [Aca-Py Admin](http://localhost:8000/docs)
+- [Aca-Py Multitenant](http://localhost:8000/docs)
+- [Webhooks](http://localhost:3010/docs)
+- [Trust registry](http://localhost:8001/docs)
+
+In order to change specific varibales and environment parameters for each container internally you likely want to do that in the respective `.env` files. Those are localted in subfolders named correspondingly to the containers within the `environments` folder. For example,
+if you want to set `auto-accept invites` for the aca-py multitennat instance form its default value `false` to `true` you can do so by changing `ACAPY_AUTO_ACCEPT_INVITES` to `true` within `environments/yoma-multitenant/aca-py-agent.default.env`. Corollary, you can do so for other vars and other containers. Stop the container(s) accordingly and start them back up using docker-compose.
```

Comment:
I'm not sure we should introduce the option to tweak e.g. the auto accept variables. The stack is currently build on auto acceptance (which we pass when creating /receiving an invitation) so auto acceptance will always be true for this.

We should probably make a distinction which environment variables are meant to be configured, and which aren't. For those that are configurable we should add some documentation, and for the other we should make it very clear in the env files they are not configurable

Reply:
Fair point! To be honest I was a little more short-sighted when writing this bit and just picked any value (not caring about potentially breaking anything). That was based on the premise that we provide the settings as is (working) and showing where and how things can/should be changed (eg as opposed to in the code). If that breaks things I thought let it break and dev X can open issue Y for it. 

Your approach obviously makes sense also will change to that if desired